lib/matplotlib/backends/backend_pgf.py

```diff
@@ -987,4 +1007,215 @@ def _cleanup_all():
     LatexManager._cleanup_remaining_instances()
     TmpDirCleaner.cleanup_remaining_tmpdirs()
 
+
 atexit.register(_cleanup_all)
+
+
+class PdfPages(object):
```

Comment:
Since we're all Python 3 now, there's no need to derive classes from `object`. It doesn't hurt, either.